Resources/views/CRUD/list_array.html.twig

```diff
@@ -2,7 +2,7 @@
 
 This file is part of the Sonata package.
 
-(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>, Toni Cornelissen <sonata-project@technetium.be>
```

Comment:
I thought it was useful having my contact information in the header. I've removed it now.